glusterapi/georep.py

```diff
@@ -111,15 +202,28 @@ def georep_checkpoint(self, mastervol, remotehost,
         return self.georep_set(mastervol, remotehost, remotevol,
                                "checkpoint", "now", remoteuser)
 
-    def georep_status(self, mastervol=None, remotehost=None,
-                      remotevol=None, remoteuser=None):
-        """
-        Geo-replication Session Status
-
-        :param mastervol: (string) Master Volume Name
-        :param remotehost: (string) Remote Host
-        :param remotevol: (string) Remote Volume
-        :param remoteuser: (string) Remote User
-        :raises: GlusterAPIError or failure
-        """
-        pass
+    def georep_status(self, mastervol=None, remotehost=None, remoteport=None,
```

Comment:
Remove remoteport for now. and only implement else part now.

Once Geo-rep list/info API is implemented, then that can be used to get master volume id and Remote volume ID.

Reply:
This API is already there if we can list geo-rep session by```HTTP.GET``` ```/v1/geo-replication```